### DISCUSSION TITLE
Add a way to get Pink Petals

### DISCUSSION THREAD

**Discussion by catter1** [Stardust-Labs-MC/StardustMC#13] [Ideas]
Players cannot currently obtain pink petals. Add a method to get them! They are very pretty :)

**Comment by ReiTheGoat01** [comment]
I think they can already be obtained from traveling merchants just like the sapling.

**Reply by aisukuma** [reply]
Yes, I’ve added that to the wandering trader trades

**Comment by aisukuma** [comment]
Server will be updating to 1.20.1
